app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/BecomeLeadController.scala

```diff
@@ -93,9 +93,10 @@ class BecomeLeadController @Inject() (
     logger.info("BecomeLeadController.postConfirmEmail")
     handleConfirmEmailPost[BecomeLeadJourney](
       previous = routes.BecomeLeadController.getConfirmEmail.url,
```

Comment:
should previous and current have the same value? looks odd to me (i might be missing something)

Reply:
Yes, good spot and I think this is actually the location of that existing bug related to the back link! have fixed this so you should see that reflected in latest commit 👍🏻